### ISSUE TITLE
[FEAT] Use bookmarks to store links

### ISSUE THREAD

**Issue by vladstudio** [willbchang/chrome-read-later#139]
**Is your feature request related to a problem? Please describe.**
Sometimes I want to access links from my "Read later" list when I am on my phone, not computer. 

**Describe the solution you'd like**
Create a special "Read later" folder with bookmarks and use it to store links. This way, Chrome for Android will sync the items.

Thanks!

**Comment by willbchang** [comment]
Hi @vladstudio, thanks for your feedback!

Sorry for the late reply, I was working on https://github.com/willbchang/chrome-read-later/releases/tag/V5.0.0 (It's available in the Chrome Web Store now).

It's surely better if we can access the "Read later" list from mobile phone, but I found it is not easy after doing some research.

1. The bookmark way surely works, but I'd like to keep the reading list and bookmark separately.
   I think:
	1. Bookmark is the thing that we don't change a lot, but access frequently(or sometimes). It takes steps to delete the unused bookmark item, otherwise it will become very very big, things can be hard to find at that time. Manage bookmark could be suffering!
	2. Reading list is the thing that we change and access frequently, it's deleted when opening it, just save the link and read it later.(I introduced local history mode for a backup in V5.0.0)

2. The build in reading list between Desktop Chrome and Mobile Chrome is separately,  I will definitely use reading list's api for "Read Later" when it's sync.
  (You can enable it by accessing `chrome://flags/#read-later`)
<img width="1392" alt="Screen Shot 2020-12-26 at 9 10 42 PM" src="https://user-images.githubusercontent.com/14329786/103152247-3053f180-47c1-11eb-988c-2c80b5883dbe.png">

<img width="300" src="https://user-images.githubusercontent.com/14329786/103152274-7446f680-47c1-11eb-8f8f-db63d8f809e0.PNG">


3. Programmingly, it's possible to
	1. Auth the github account, export it to github page automatically, you can access the reading list via github page on the phone.
	2. Share it from an owned domain, like [OneTab](https://chrome.google.com/webstore/detail/onetab/chphlpgkkbolifaimnlloiipkdnihall?hl=en).
   
   Both of them requires a lot of work, and it's not native.

4. Try [Kiwi Browser](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser&hl=en_US), it seems [able to use chrome extension on Android](https://play.google.com/store/apps/details?id=com.kiwibrowser.browser&hl=en_US), sorry that I cannot test it for you because I'm using iPhone.


Thanks again for bringing this feature! I'll keep eyes on it.

**Comment by vladstudio** [comment]
@willbchang - thank you for your reply! I totally agree it's better to use Chrome's internal reading list's api. It's actually a bit sad to see "read later" functionality built inside Chrome - this inevitably means your extension will be less used. The idea to close a tab when I add it to "read later" and remove an item from "read later" when I open it is genius :-)

**Comment by willbchang** [comment]
>It's actually a bit sad to see "read later" functionality built inside Chrome - this inevitably means your extension will be less used. 

I agree with you, I have thought about making it independently. I'll let you know at that time. 

>The idea to close a tab when I add it to "read later" and remove an item from "read later" when I open it is genius :-)

I tried almost all the read later extensions in Chrome web store and none of them did this, that's why I created it. XD


I'll keep this issue open, because I think it can be solved in the future. ;)

**Comment by GeoNomad** [comment]
I have forked a version which posts the URLs to a web app that can be used on a phone with any browser.

A [demo php webapp](https://readlater.000webhostapp.com/) is available.

Just a bit of a hack to get the functionality I wanted but if you have a server available to you, you can extend it as you wish. The demo is on a free server I have never tried before, but the price is right.

https://github.com/GeoNomad/chrome-read-later

**Comment by willbchang** [comment]
@GeoNomad Great work! This will solve the problem for those who want to access the reading list from the phone.

**Comment by willbchang** [comment]
https://developer.chrome.com/blog/extension-news-october-2023/#coming-soon-apis
<img width="661" alt="image" src="https://github.com/willbchang/chrome-read-later/assets/14329786/fc6620af-c6df-4e8b-8374-384573946332">

From Chrome 120, this extension should be able to use **ReadingList API** to sync the reading list between phone and  computer.

- [Issue 1421058: [ Extensions ] Reading List API](https://bugs.chromium.org/p/chromium/issues/detail?id=1421058)
- [Temporary Documentation](https://docs.google.com/document/d/1f1wW2955nY5Rp3vfS0JGuo1kACOAmsjZupvLYknWKII/edit)
<img width="660" alt="image" src="https://github.com/willbchang/chrome-read-later/assets/14329786/bf3112da-ea77-4c8d-a19a-eafc1241719b">
